<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QDialog" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loginwindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 11pt &quot;等线&quot;;
}

QLabel#login_picturelabel{
border-image: url(:/image/nightsky.jpg);
font: 20pt &quot;等线&quot;;
color: rgb(255, 255, 255);
}

QPushButton#login_registerbutton{
color: rgb(38, 133, 227);
}

QPushButton#login_helpbutton{
color: rgb(38, 133, 227);
}

QPushButton#login_loginbutton{
border: 1px solid #b8b8b8;
border-radius: 2px;
background-color: #b8b8b8;
}

QPushButton#login_loginbutton:enabled{
color: rgb(255, 255, 255);
border: 1px solid #384299;
border-radius: 2px;
background-color: #384299;
}

QPushButton#login_loginbutton:pressed{
color: #34fcfc;
background-color: #5d70ff;
}

QPushButton#login_exitbutton{
color: rgb(255, 255, 255);
border: 1px solid #384299;
border-radius: 2px;
background-color: #384299;
}

QPushButton#login_exitbutton:pressed{
color: #34fcfc;
background-color: #5d70ff;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="login_picturelabel">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>150</height>
      </size>
     </property>
     <property name="text">
      <string>导航控制信息系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="login_numberlabel">
           <property name="text">
            <string>工号：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="login_numbertext">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
           <property name="placeholderText">
            <string>工号-数字开头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="login_registerbutton">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string> 注册</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="login_passwordlabel">
           <property name="text">
            <string>密码：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="login_passwordtext">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>密码-最长10位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="login_helpbutton">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string> 帮助</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="login_loginbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="login_exitbutton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>login_numbertext</tabstop>
  <tabstop>login_passwordtext</tabstop>
  <tabstop>login_registerbutton</tabstop>
  <tabstop>login_helpbutton</tabstop>
  <tabstop>login_loginbutton</tabstop>
  <tabstop>login_exitbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
